components/_patterns/01-atoms/04-images/icons/_icon.twig

```diff
@@ -8,10 +8,9 @@
  */
 #}
 {% set icon_base_class = icon_base_class|default('icon') %}
-{# For Drupal: Update this path to match your theme directory structure #}
 {# For Pattern Lab: The icon_url is defined in _data/paths.json. No action is required #}
-{% set icon_url = icon_url|default('/themes/custom/YOURTHEME/dist/img/sprite/') %}
-{# Update `emulsify` here to your theme's machine name #}
```

Comment:
@shaal I am curious as to the broken behavior you experienced here.  Did "emulsify" on this line get replaced?  On the line right after this did it not get replaced?  I am definitely pro simplifying install processes so I would love to hear more about the bug you encountered.

Reply:
I think the goal here is to set this as `emulsify` by default so that it can be auto-replaced. In the "before" code, it says to manually enter your theme's machine name. Setting it automatically on cloned theme creation is preferred.

Reply:
Created this ticket for follow up on this: https://github.com/fourkitchens/emulsify/issues/271